.pipelines/build-pipelines.yml

```diff
@@ -39,15 +39,23 @@ steps:
 # We cannot set this in variables section above because $(isNugetRelease) 
 # is not available at pipeline compilation time.
 - bash: |
-    echo ISNUGETRELEASE = $ISNUGETRELEASE
+    echo IsNugetRelease = $ISNUGETRELEASE
+    echo IsReleaseCandidate = $ISRELEASECANDIDATE
+    dabVersion=$(major).$(minor).$(patch)
     if [ "$ISNUGETRELEASE" = "true" ]
     then
-      dabVersion=$(major).$(minor).$(patch)
+      if [ "$ISRELEASECANDIDATE" = "true" ]
```

Comment:
nit: Re-naming $ISRELEASECANDIDATE to something like `isPreRelease` would make it easier to follow (intuitively distinguishes pre-release vs release)

Reply:
I did it specifically because we are calling it release candiate version, with `-rc` tag. Also, when I looked for differences between the two, I found out the main differentiating factor is that the `preRelease` version may contain bugs and may not have all feature implemented. It refers to the version still under development.
While, the `releaseCandidate` version refers to a version of software that is considered to be near completion and ready for release. It has undergone rigorous testing and is expected to be stable, but may still undergo some final adjustments before the official release.

So, `releaseCandidate` seems like a suitable choice here.